_config/ispn.yml

```diff
@@ -50,16 +50,16 @@ stable:
 # Please remove or comment out the following 'unstable' version section if this is not applicable at the moment.
 unstable:
   minor_version: 12.0
-  version: 12.0.0.Dev01
-  maven_latest: 12.0.0.Dev01
-  release_date: 2020/07/27
-  fixed_issues: https://issues.redhat.com/secure/ReleaseNote.jspa?projectId=12310799&version=12346555
+  version: 12.0.0.Dev03
+  maven_latest: 12.0.0.Dev03
+  release_date: 2020/09/8
+  fixed_issues: https://issues.redhat.com/secure/ReleaseNote.jspa?projectId=12310799&version=12347011
   codename: N/A
-  whats_new: Hibernate Search 6 + Lucene 8, console improvements, etc...
+  whats_new: Documentation improvement, Cross Site Replication Conflict Resolutionn, Console improvements
```

Comment:
I think this is "what's new in ISPN 12" and not in the release.

Reply:
reverted